RELEASE_NOTES.md

```diff
@@ -1,37 +1,43 @@
-<!-- - This file is reset every time a new release is done. The contents of this file are for the currently unreleased version. Example Note: ## Example Heading Details about the thing that changed that needs to get included in the Release Notes in markdown. -->
+# Ohai Release Notes 13.2:
 
-# Ohai Release Notes 13.1:
+Ohai 13.2 has been a fantastic release in terms of community involvement with new plugins, platform support, and critical bug fixes coming from community members. A huge thank you to msgarbossa, albertomurillo, jaymzh, and davide125 for their work.
 
 ## New Features
 
-### Shellout now injects additional paths
+### Systemd Paths Plugin
 
-The shell_out helper used by Ohai plugins to run system commands has been updated to inject commonly used paths when shelling out. This allows us to find system utilities even when the user's PATH variable lacks common locations such as /sbin.
+A new plugin has been added to expose system and user paths from systemd-path (see <https://www.freedesktop.org/software/systemd/man/systemd-path.html> for details).
 
-### EC2 metadata improvements
+### Linux Network, Filesystem, and Mdadm Plugin Resilience
 
-The EC2 plugin and helpers have been updated to support the latest versions of Amazon's EC2 metadata endpoint, which includes new IPV6 data. Additionally, when available, we will reuse HTTP connections to reduce load on metadata servers and reduce the memory footprint of Ohai. This is particularly useful in the Openstack plugin which uses the EC2 helpers and connects to servers that support connection reuse.
+The Network, Filesystem, and Mdadm plugins have been improved to greatly reduce failures to collect data. The Network plugin now better finds the binaries it requires for shelling out, filesystem more gracefully falls back to secondary data sources, and mdadm handles arrays in bad states.
```

Comment:
actually filesystems always uses multiple data sources now, instead of failing back